docs/conf/portland/2018/about.rst

```diff
@@ -1,19 +1,14 @@
-:template: 2018/na/generic.html
-:banner: _static/2018/assets/headers/team.png
-
+:template: {{year}}/{{templatecode}}/generic.html
+:banner: _static/{{year}}/assets/headers/team.png
```

Comment:
Should we be segmenting these by year? I think having generic community photos works fine, and feels like extra complexity we don't need.